pkg/controllers/machine_controller.go

```diff
@@ -268,35 +265,29 @@ func (r *MachineReconciler) startCheckAndEnqueueVolumeResize(ctx context.Context
 }
 
 func (r *MachineReconciler) startCheckAndEnqueueMachineState(ctx context.Context, log logr.Logger) {
```

Comment:
A more suitable name for this function could be `monitorLifecycleEvents` or `watchLifecycleEvents`

Reply:
i prefer keep the same name convention. maybe `startMonitorLifecycleEvents`

Reply:
How about `startEnqueMachineByLibvirtEvent`? Because that is essentially what is going to happen here.